addons/web/controllers/session.py

```diff
@@ -50,7 +50,7 @@ def authenticate(self, db, login, password, base_location=None):
                 http.root.session_store.rotate(request.session, env)
                 request.future_response.set_cookie(
                     'session_id', request.session.sid,
-                    max_age=http.SESSION_LIFETIME, httponly=True
+                    max_age=env["ir.config_parameter"].get_param("sessions.max_inactivity_seconds", http.SESSION_LIFETIME)
```

Comment:
did you test it? is it working without `int()`?!
```suggestion
                    max_age=int(env["ir.config_parameter"].get_param("sessions.max_inactivity_seconds", http.SESSION_LIFETIME))
```